Source/Moya+Defaults.swift

```diff
@@ -3,8 +3,12 @@ import Alamofire
 /// These functions are default mappings to `MoyaProvider`'s properties: endpoints, requests, manager, etc.
 public extension MoyaProvider {
     public final class func DefaultEndpointMapping(target: Target) -> Endpoint<Target> {
-        let url = target.baseURL.URLByAppendingPathComponent(target.path).absoluteString
-        return Endpoint(URL: url, sampleResponseClosure: {.NetworkResponse(200, target.sampleData)}, method: target.method, parameters: target.parameters)
+        if let url = target.baseURL.URLByAppendingPathComponent(target.path) {
```

Comment:
Cool, much better! What do you think about this:

```swift
guard let url = target.baseURL.URLByAppendingPathComponent(target.path),
      let stringURL = url.absoluteString else {
    fatalError(...)
}

return Endpoint(URL: stringURL, sampleResponseClosure: {.NetworkResponse(200, target.sampleData)}, method: target.method, parameters: target.parameters)
```

This keeps the "happy path" unindented, what do you think?